lib/exprengine.cpp

```diff
@@ -1387,13 +1387,13 @@ bool ExprEngine::IntRange::isLessThan(DataBase *dataBase, int value) const
 
 bool ExprEngine::FloatRange::isEqual(DataBase *dataBase, int value) const
 {
-    const Data *data = dynamic_cast<Data *>(dataBase);
-    if (data->constraints.empty())
-        return true;
     if (MathLib::isFloat(name)) {
```

Comment:
@danmar I suppose we don't need to involve z3 solver, when we known we have a floating-point literal. In this case, we can't have any constraints, so I'm ignoring them here.